### PR TITLE
Fix Ticker cannot stop for releasing resource

### PR DESCRIPTION
**Pull request by AlexiaChen**
Ticker Need to use like this: https://github.com/harmony-one/harmony/blob/b602e8d454694373ab8f68b4e95cfd3a60670bb9/core/tx_pool.go#L317-L324

Do not forget 

```go
defer ticker.Stop()
```

if forget do such thing, maybe leading this meory leak like this: 
- https://github.com/google/goexpect/issues/3
- https://github.com/google/goexpect/pull/4

### CODE REVIEW

File: rpc/filters/api.go [hunk 1]
```diff
@@ -67,6 +67,7 @@ func (api *PublicFilterAPI) isEth() bool {
 func (api *PublicFilterAPI) timeoutLoop() {
 	// TODO ek – infinite loop; add shutdown/cleanup logic
 	ticker := time.NewTicker(5 * time.Minute)
+	defer ticker.Stop()
```
Comment (JackyWYX) [inline rpc/filters/api.go:70]: Do you think this can be the potential memory leak reason?

**Review by JackyWYX — APPROVED** [review]
Thanks for pointing that out. That were definitely some wrong usage of ticker. Correcting them shall be great.

**Review by LeoHChen — APPROVED** [review]
I felt this may contribute to the memory leakage

File: rpc/filters/api.go (hunk 1 above)
Comment (AlexiaChen) [reply rpc/filters/api.go:70]: I just corret them all, Some of this code will not have memory leaks because it is an infinite loop and does not exit.

**Pull request merged**